css: prettified bounty_details page

thelostone-mc opened the pull request:
##### Description

As per earlier discussions where a decision was made to use `OpenSans`, I figured we'd start slow by migrating in batches. Would make it easier to review + refactoring happens along the way

- removed inline css
- used bootstrap grid system
- added OpenSansLight font
- optimized for mobile view
- replaced old newsletter with new one

<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->


<!-- A description on what this PR aims to solve -->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, ui, crypto, etc). -->
- ui

owocki commented:
@thelostone-mc thanks for the innitiative here!

![screencapture-localhost-8080-legacy-funding-details-1517326173012](https://user-images.githubusercontent.com/513929/35574686-b6855470-0597-11e8-879a-ae7fd4c5bd85.png)
![screencapture-localhost-8080-legacy-funding-details-1517326217812](https://user-images.githubusercontent.com/513929/35574726-d019bfac-0597-11e8-9237-1943e1e7aea4.png)


@pixelantdesign what do you think of this?

my initial thoughts: 
* lots of nice little clean up :)
* not crazy about the white background / center-ness on the email capture form.
* tablet view could use some left padding

thelostone-mc commented:
@owocki So when I had coded up the toolbox page -> as set up by @algae12 I figured we'd eventually adopt this throughout the site!


<img width="1440" alt="screen shot 2018-01-30 at 9 55 39 am" src="https://user-images.githubusercontent.com/5358146/35582456-bec4d91a-05a3-11e8-922a-2edc29153bf6.png">

- we could give it a background or just a border below it 
- the tablet view did look alright on firefox , I'll double check it

thelostone-mc commented:
@owocki fixed the tablet view : Also the submitter profile is visible on mobile!

owocki commented:
personally id vote for left aligning it and adding abackground.  curious what @pixelantdesign thinks

thelostone-mc commented:
I'll make the change if that's the popular vote

PixelantDesign commented:
I think the layout clean up is a huge improvement! For right now, I think we should keep Futura until we look at the brand holistically. I do agree that the new find out about funded issues makes the top part seem empty and unbalanced. A background color would help to anchor that area. There is enough room across the top so that the information doesn't need to be stacked or centered. Thoughts?

thelostone-mc commented:
@PixelantDesign pushing it to the bottom wouldn't be smart would it?  🤔 
Also if it's futura , do we also change the toolbox page ?

PixelantDesign commented:
No, I don't think we should move it to the bottom. I would leave it how it is for now. Thanks!

thelostone-mc commented:
@mbeacom review needed. Rebased and rewrote stuff to resolve conflicts 😅  

@PixelantDesign , @owocki  I need suggestions on color for the background. Plain white looks the nicest to me still 

Screenshots (mobile + tablet + desktop ) :


![screenshot-2018-2-1 test ether facuet thelostone-mc funded issue detail gitcoin](https://user-images.githubusercontent.com/5358146/35714983-b169a826-0784-11e8-9961-0f4733e403c2.png)


![screenshot-2018-2-1 test ether facuet thelostone-mc funded issue detail gitcoin 1](https://user-images.githubusercontent.com/5358146/35714984-b196c23e-0784-11e8-90d7-8728b0e177d3.png)


![screenshot-2018-2-1 test ether facuet thelostone-mc funded issue detail gitcoin 2](https://user-images.githubusercontent.com/5358146/35714985-b1b377a8-0784-11e8-8146-4f2db7852e30.png)


![screenshot-2018-2-1 test ether facuet thelostone-mc funded issue detail gitcoin 3](https://user-images.githubusercontent.com/5358146/35714986-b1cdfa56-0784-11e8-8a2c-b0903ea3ba51.png)


![screenshot-2018-2-1 test ether facuet thelostone-mc funded issue detail gitcoin 4](https://user-images.githubusercontent.com/5358146/35714987-b1e84532-0784-11e8-8731-b850322d7c21.png)

owocki commented:
looks great :)

i still think that having a background color for the email subscribe module would be better.. but if im on the minority there... happy to relent

owocki commented:
i also like @thelostone-mc 's idea to

> either shifting the newsletter to the bottom (like on /toolbox) on top (like in every other page )

thelostone-mc commented:
@owocki I'm okay with adding a background color but could use some help in deciding on one ! 

If we were pushing it down , I would strongly support a white background :P

PixelantDesign commented:
I'm good with moving the email area to the bottom since it's consistent with the toolbox page. I don't think a background is needed though. Perhaps just a 1px rule above it?

owocki commented:
what @pixelantdesign said

thelostone-mc commented:
@owocki , @mbeacom  : changes done + conflicts resolved ! review/merge ready ^_^

![10f98000bac13ec26f5aebf9f2b2bf1e41a3104a9893f73dde9dd76cbce9daff](https://user-images.githubusercontent.com/5358146/35757136-a7ea66c0-0822-11e8-80b6-c705ff8a76ab.jpg)

owocki commented:
sorry for the radio silence.. was traveling the last 9 days.  

@PixelantDesign updated bounty page details below:  any design feedback?

![screencapture-localhost-8080-funding-details-1518474207913](https://user-images.githubusercontent.com/513929/36123047-b1ac3362-1008-11e8-99e0-1481ccc9d584.png)

my 002 wei: seems to me that there should be some padding near the top.. the page content is too close to the toolbar

owocki commented:
i think it might be worth hiding the email subscription module from the loading page also

![screencapture-localhost-8080-funding-details-1518474322155](https://user-images.githubusercontent.com/513929/36123124-fb141466-1008-11e8-846d-425a50e28d93.png)

jasonrhaas commented:
Agree with the padding comment.  It looks like some solid UI improvements.  Thanks @thelostone-mc!  I have some changes I made to the bounty_details.html for multiple bounties so once this is merged into `master` I will want to pull this in the `february_integration_branch`.

thelostone-mc commented:
@owocki  Ah shit forgot that !! Fixed it ^_^

@jasonrhaas Yeah I figured you would need this in `february_integration_branch`, hence wrapped it up quick 


<img width="1440" alt="screen shot 2018-02-13 at 7 59 45 am" src="https://user-images.githubusercontent.com/5358146/36130789-a394d1b2-1094-11e8-862c-f7d202bbfde1.png">
<img width="1440" alt="screen shot 2018-02-13 at 8 00 17 am" src="https://user-images.githubusercontent.com/5358146/36130799-abdcc636-1094-11e8-91a0-c9d2bb5eedd5.png">

owocki commented:
@thelostone-mc i hate to do this to you, but can you resolve the merge conflicts one last time?

thelostone-mc commented:
@owocki @mbeacom @jasonrhaas Resolved Good to go ^_^

thelostone-mc commented:
@owocki give me a heads up whenever you are ready to get this merged! Will hold off on resolving conflicts till then rather than doing it multiple times  -.-

They say third time's the charm. So maybe ? :P

thelostone-mc commented:
@owocki conflicts. not cool. that's just cheating :P

owocki commented:
argh.. sorry.. travel has made me an unreliable steward of my inbox.  this
is near the top of my list for this week.

On Thu, Feb 22, 2018 at 9:03 AM, Aditya Anand M C <notifications@github.com>
wrote:

> @owocki <https://github.com/owocki> conflicts. not cool. that's just
> cheating :P
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gitcoinco/web/pull/323#issuecomment-367728794>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAfXiVflU3vL4ZasNr8hDuEzHOAtus2aks5tXY_EgaJpZM4Rxy6l>
> .
>



-- 


@owocki <http://www.twitter.com/owocki>

_________________________________________

check out what i'm building on github <http://github.com/owocki>

or what i'm shooting on photoshelter <http://photography.owocki.com/>

or find me on twitter <http://www.twitter.com/owocki>, facebook
<http://www.facebook.com/kevinowocki>, instagram
<http://instagram.com/owocki>, linkedin <http://www.linkedin.com/in/owocki>,
and owocki.com <http://www.owocki.com/?utm_source=emailsig>.

see you around teh internets!

PS -- Come to the next Boulder Blockchain
<https://www.meetup.com/Boulder-Blockchain/> meetup.  Be there or be ^ 2.
PPS -- Have I mentioned that *G**itcoin is live?*  More @ https://gitcoin.co
